systest/bgindex/docker-compose.yml

```diff
@@ -19,109 +19,8 @@ services:
       source: ../../ee/acl/hmac-secret
       target: /secret/hmac
       read_only: true
-    command: /gobin/dgraph alpha --my=alpha1:7080 --zero=zero1:5080,zero2:5080,zero3:5080
```

Comment:
So this PR has 3 changes?
1. closer fix
2. Assign UID fix
3. bgindex fix?

Reply:
yes, this PR has above three fix. let me change the description also